baseplate/context/cassandra.py

```diff
@@ -93,8 +94,10 @@ def execute_async(self, query, parameters=None, timeout=_NOT_SET):
         # TODO: include custom payload
         if isinstance(query, six.string_types):
             span.set_tag("statement", query)
-        elif isinstance(query, SimpleStatement):
+        elif isinstance(query, SimpleStatement, PreparedStatement):
```

Comment:
I think this is invalid syntax: `isinstance` only takes two params though the latter can be a tuple of types.